app/models/tasks/copy_transcript_task.rb

```diff
@@ -0,0 +1,47 @@
+class Tasks::CopyTranscriptTask < ::Task
+  def transcript
+    owner
+  end
+
+  def source_url
+    transcript.href
+  end
+
+  def porter_tasks
+    [
+      {
+        Type: "Inspect"
+      },
+      {
+        Type: "Copy",
+        Mode: "AWS/S3",
+        BucketName: ENV["FEEDER_STORAGE_BUCKET"],
+        ObjectKey: porter_escape(transcript.path),
+        ContentType: "REPLACE",
+        Parameters: {
+          CacheControl: "max-age=86400",
+          ContentDisposition: "attachment; filename=\"#{porter_escape(transcript.file_name)}\""
+        }
+      }
+    ]
+  end
+
+  def update_transcript
+    transcript.status = status
+
+    if complete?
+      info = porter_callback_inspect
+      transcript.size = porter_callback_size
+
+      # only return for actual images - not detected images in id3 tags
+      if info[:Transcript] && porter_callback_mime&.starts_with?("transcript/")
```

Comment:
Hmm... I'm not sure this is going to exist.  At least no yet - Porter `Inspect` knows things about [audios/videos/images](https://github.com/PRX/Porter/blob/main/src/lambdas/inspect/index.js#L44-L46), but nothing about transcripts.